lib/redact-secrets/is-secret.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+// This module is a fork of
+// https://github.com/watson/is-secret/blob/563cce48b4b89fd3f99221432a3f37b4662672b0/index.js
```

Comment:
```suggestion
// https://github.com/watson/is-secret/blob/v1.2.1/index.js
```

*Could* use the tag name.

Reply:
#same as above.